### PR TITLE
Mention new Discord server invite link

### PR DESCRIPTION
**Pull request by bittner**
The IRC channel #pocoo doesn't seem to be used anymore. When you login there, a friendly notice mentions the "New Home on Discord".

This change updates the related information, so interested parties don't have to do the unnecessary hop to IRC first.

### CODE REVIEW

**Comment by ThiefMaster** [comment]
AFAIK we have the vanity URL https://discord.gg/pallets people can use to join. I'd also keep the mention of the IRC channel since we have a relay bot and I'd like to not exclude people who prefer IRC and don't even have Discord. (but we should be consistent among the different pallets repos)

**Comment by davidism** [comment]
I'll change this for now, but I need to copy over the work we did on the Werkzeug templates.

**Comment by bittner** [comment]
I tried to address your comment. Does that now look like you'd want to have it?

**Comment by ThiefMaster** [comment]
I'd reword it to put the discord server first (it's the preferred place for people to go) and just keep a mention of the IRC channel after that.

cc @davidism in case he prefers some text that's the same everywhere

**Comment by davidism** [comment]
Thanks for working on this, but I should really just copy over the new templates from Werkzeug.

**Comment by bittner** [comment]
Alright, I understand. Copied the templates over from Werkzeug. :checkered_flag:

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ jobs:
           - {name: 'PyPy', python: pypy3, os: ubuntu-latest, tox: pypy3}
           - {name: Style, python: '3.8', os: ubuntu-latest, tox: style}
           - {name: Docs, python: '3.8', os: ubuntu-latest, tox: docs}
+          - {name: Typing, python: '3.8', os: ubuntu-latest, tox: mypy}
```
Comment (davidism) [inline .github/workflows/tests.yaml:25]: This shouldn't be here.

Comment (bittner) [reply .github/workflows/tests.yaml]: Okay, done. Anything else? Because, really, I just copied the entire folder and did search+replace for "Werkzeug".
